lib/acl.c

```diff
@@ -336,6 +337,7 @@ init_ipv4_acls(struct gatekeeper_if *iface)
 	iface->ipv4_acls.funcs[ACL_NO_MATCH] = drop_unmatched_pkts;
 	iface->ipv4_acls.ext_funcs[ACL_NO_MATCH] = NULL;
 	iface->ipv4_acls.func_count = 1;
+	iface->ipv4_acls.enabled = true;
```

Comment:
BUG: `init_ipv4_acls()` is only called if `ipv4_acl_enabled()` returns `true` in `start_iface()`. But `ipv4_acl_enabled()` only returns `true` if `init_ipv4_acls()` is called once.

Address the same problem for `init_ipv6_acls()` as well.

Reply:
Ignore this comment. `ipv4_pkt_filter_add()` is calling `init_ipv4_acls()`.